src/main/java/com/epam/usermanager/repository/DeviceRepository.java

```diff
@@ -0,0 +1,202 @@
+package com.epam.usermanager.repository;
+
+import com.epam.usermanager.entity.Device;
+import com.epam.usermanager.entity.Type;
+import com.epam.usermanager.entity.User;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DeviceRepository {
+    String USERNAME = "root";
```

Comment:
Read about java constants (private/public static final) and change those fields to constants